.pre-commit-config.yaml

```diff
@@ -2,24 +2,28 @@ repos:
   - repo: https://github.com/psf/black
     rev: 20.8b1
     hooks:
-    - id: black
-      args: [--target-version=py36]
-      language_version: python3
+      - id: black
+        args: [--target-version=py36]
+        language_version: python3
   - repo: https://github.com/PyCQA/isort
     rev: 5.6.4
     hooks:
-    - id: isort
-      language_version: python3
+      - id: isort
```

Comment:
Extra sequence indentation looks weird

Reply:
The style is quite conventional and common. Beyond being the standard formatting of prettier, it is also the formatting used by the popular Ansible project. For example: https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html